src/utilities/clipboard-utils.ts

```diff
@@ -0,0 +1,21 @@
+import { gImageMap} from "../models/image-map";
+type OnComplete = (params: File | string) => void;
+
+export const pasteClipboardImage = async (callback: OnComplete, mode: string) => { //mode either "file" or "url"
```

Comment:
we need some argument to tell `pasteClipboardImage` how to invoke our callback which is different in `image-tool.tsx `and `drawing-tool.tsx`. What are your thoughts on this? 

I didn't add a type to mode, I could do that on line 3 with 
type mode = "url" | "file"